src/App.css

```diff
@@ -1,24 +1,72 @@
-.App {
+.main {
+  background-color: #ff5698;
+}
+
+h1,
+h2 {
   text-align: center;
+  padding: 0.5rem;
+  padding-top: 0;
+  margin-top: 0;
+}
+
+h1,
+h2,
+h4 {
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+
+img {
+  margin-left: 1rem;
+}
+
+.back {
+  max-width: 25px;
+  height: auto;
 }
```

Comment:
- [ ] Kindly make sure to align your `back` button correctly in order to match the provided design.


    - ### YOUR DESIGN
    
     ![image](https://user-images.githubusercontent.com/87197412/233772322-a651975b-d852-4bb5-9ac0-b8e836b8f7d5.png)

    - ### PROVIDED DESIGN
    
    ![image](https://user-images.githubusercontent.com/87197412/233772310-dd2632f8-f200-44c3-8804-4a17452b061d.png)